src/server/socketHandlers/mwPresenceSubscribe.js

```diff
@@ -0,0 +1,30 @@
+import parseChannel from './parseChannel';
```

Comment:
presence subscriptions are handled in the pubsub layer, which is lower than the data transport layer. That means we have to drop down a level to shut em down.